client/src/components/Shop/style.css

```diff
@@ -1,6 +1,5 @@
 #addProductBtn {
```

Comment:
these could probably be classes instead of id's. are these buttons really going to be unique? In the future, you could have the same button style. Who knows. Keep that in mind. Unless you really know that this 1 button is going to only exist in 1 place then it's fine.